### DISCUSSION TITLE
Add single glyph

### DISCUSSION THREAD

**Discussion by chkone** [aiekick/ImGuiFontStudio#15] [Ideas]
As some service give us the possibility to download single glyph for example:
https://thenounproject.com/search/?q=transform
Does that make sense ImGuiFontStudio that allow to push_back a single glyph to an existing font?
The sources give us png or svg. That need some setup, like bounding box, origin, ...

For example when we work with contractor to get icons they can provide *.svg. but they rarely work on direct *.ttf.

**Comment by aiekick** [comment]
hello,

thanks.

its planned at some stage to have the ability to import svg file in the font. like the creator of these font are doing.
currently im writing my ttf reader/writer for have the ability to merge font easily (not good at all for the moment) and write colored glyph.
after i will finalize,the glyph editor for create / modify glyph and after will work on svg import :)
